### DISCUSSION TITLE
Accuracy with Speech to Text and Start/End Times

### DISCUSSION THREAD

**Discussion by arciisine** [deepgram/community#629] [General help]
I'm currently working with a video, and I'm noticing that the even with the enhanced model, I'm getting timestamps for sentences that are eating the end of words.  One of the example I'm currently looking at is producing a sentence time frame that is missing the last ~500ms of audio.  

When running the same audio through Amazon's transcription service the time boundaries are spot on, and so this looks to be a discrepancy.  I am happy to share the file if necessary/desired.  Its a 5 minute subset of https://www.youtube.com/watch?v=oM2h3KnWAWY . 

The real question here, is there some guidance on how to best adapt/adjust the timings to better fit to the source content (e.g. always add 100ms of padding, etc.), or is this a known limitation of with respect to the transcription accuracy?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @arciisine, what are the models you've tested so far? You mention Enhanced; do also try our newest model Nova-2 (`model=nova-2`). Some customers also prefer Base (`model=base`) for finer-grained timestamps, even though the word error rate is slightly lower as it's an older model. We have customers using Deepgram transcript timestamps for captioning, video processing, etc, so relying on timestamps is a common use case. Please let me know if you try a few of our models and are still finding that none of them are meeting your needs, and we can go deeper.

**Reply by arciisine** [reply]
So I tried all the models.  I believe base was more accurate on timings, but the quality of the transcription deteriorated and many words were not understood.

I'll respond during the week, but can provide the input and some sample timings were things are off.
